jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/KtorClient.kt

```diff
@@ -98,12 +98,20 @@ public open class KtorClient(
 		}.buildString()
 	}
 
+	/**
+	 * This is a temporary workaround until we have a proper way to send device names to the server.
+	 * It will filter out all special characters.
+	 */
+	private fun String.encodeAuthorizationHeaderValue() = replace("[^\\w\\s]".toRegex(), "").trim()
```

Comment:
This should probably provide a default value in the case that all characters in the name are invalid.

Reply:
![image](https://user-images.githubusercontent.com/2305178/101675465-20ff4500-3a5a-11eb-9342-bc0d4470bd73.png)
No name works fine - it will just not display anything in the dashboard, left a device without a name and right my browser session with a name.